Modules/FindCUDA.cmake

```diff
@@ -734,15 +767,14 @@ if(NOT CUDA_VERSION VERSION_LESS "5.5")
   cuda_find_library_local_first(CUDA_cudart_static_LIBRARY cudart_static "static CUDA runtime library")
   mark_as_advanced(CUDA_cudart_static_LIBRARY)
 endif()
-if(CUDA_cudart_static_LIBRARY)
-  # Set whether to use the static cuda runtime.
-  option(CUDA_USE_STATIC_CUDA_RUNTIME "Use the static version of the CUDA runtime library if available" ON)
-else()
-  option(CUDA_USE_STATIC_CUDA_RUNTIME "Use the static version of the CUDA runtime library if available" OFF)
-endif()
 
-if(CUDA_USE_STATIC_CUDA_RUNTIME)
-  if(UNIX)
+if(NOT CUDA_USE_STATIC_CUDA_RUNTIME STREQUAL OFF)
+  if(CUDA_cudart_static_LIBRARY)
+    # Set whether to use the static cuda runtime.
+    set(CUDA_USE_STATIC_CUDA_RUNTIME ON CACHE BOOL "Use the static version of the CUDA runtime library if available")
```

Comment:
Typically if I want to override a default I do one of two things.

If I want to override it locally in one specific build I add this to the configure line:

cmake ... -DCUDA_USE_STATIC_CUDA_RUNTIME:BOOL=OFF

If I want to set it for all configurations I add this:

option(CUDA_USE_STATIC_CUDA_RUNTIME "use static runtime" OFF)
find_package(CUDA)

Because the first cache variable set will be the winner, you can set it before FindCUDA can set the default.